model/users.model.js

```diff
@@ -0,0 +1,9 @@
+const db = require("../db/connection");
+
+exports.fetchUsers = () => {
+    let queryStr = 'SELECT username FROM users';
+  
```

Comment:
picky picky but you are not expecting to reassign this queryStr at a later date so make your variable a const for extra protection!